targets/targets.json

```diff
@@ -6765,7 +6765,6 @@
             "MCU_PSOC6_M4"
         ],
         "components_add": [
-            "SCL",
```

Comment:
What I believe is a similar problem was encountered a while ago in TARGET_Cypress/TARGET_PSOC6/COMPONENT_WHD. The solution in that case was to add a trivial mbed_lib.json file under COMPONENT_WHD so that it is excluded in the bare metal build process by virtue of a.)being considered a library and b.)not being included in the bare metal profile's whitelist of libraries.

Would a similar approach be feasible here? I don't think this is necessarily wrong, but it would be better to be consistent if we can.

Reply:
That sounds better, let me try it.

Reply:
Done

Reply:
Looks good, thanks